desktop-widgets/filterwidget2.ui

```diff
@@ -13,461 +13,457 @@
-        </property>
-       </item>
-      </widget>
-     </item>
-     <item row="10" column="1" colspan="2">
-      <widget class="QComboBox" name="locationMode">
-       <item>
-        <property name="text">
-         <string>All of</string>
-        </property>
-       </item>
-       <item>
-        <property name="text">
-         <string>Any of</string>
-        </property>
-       </item>
-       <item>
-        <property name="text">
-         <string>None of</string>
-        </property>
-       </item>
-      </widget>
-     </item>
-     <item row="11" column="1" colspan="2">
-      <widget class="QComboBox" name="suitMode">
-       <item>
-        <property name="text">
-         <string>All of</string>
-        </property>
-       </item>
-       <item>
-        <property name="text">
-         <string>Any of</string>
-        </property>
-       </item>
-       <item>
-        <property name="text">
-         <string>None of</string>
-        </property>
-       </item>
-      </widget>
-     </item>
-     <item row="12" column="1" colspan="2">
-      <widget class="QComboBox" name="dnotesMode">
-       <item>
-        <property name="text">
-         <string>All of</string>
-        </property>
-       </item>
-       <item>
-        <property name="text">
-         <string>Any of</string>
-        </property>
-       </item>
-       <item>
-        <property name="text">
-         <string>None of</string>
-        </property>
-       </item>
-      </widget>
-     </item>
-     <item row="13" column="1" colspan="2">
-      <widget class="QComboBox" name="equipmentMode">
-       <item>
-        <property name="text">
-         <string>All of</string>
-        </property>
-       </item>
-       <item>
-        <property name="text">
-         <string>Any of</string>
-        </property>
-       </item>
-       <item>
-        <property name="text">
-         <string>None of</string>
-        </property>
-       </item>
-      </widget>
-     </item>
-     <item row="13" column="0">
-      <spacer name="verticalSpacer">
-       <property name="orientation">
-        <enum>Qt::Vertical</enum>
-       </property>
-       <property name="sizeHint" stdset="0">
-        <size>
-         <width>20</width>
-         <height>40</height>
-        </size>
-       </property>
-      </spacer>
-     </item>
-    </layout>
-   </widget>
-  </widget>
+  <layout class="QGridLayout" name="gridLayout_2">
+   <item row="0" column="0">
+    <widget class="QScrollArea" name="scrollArea">
+     <property name="frameShape">
+      <enum>QFrame::NoFrame</enum>
+     </property>
+     <property name="frameShadow">
+      <enum>QFrame::Plain</enum>
+     </property>
+     <property name="widgetResizable">
+      <bool>true</bool>
+     </property>
+     <widget class="QWidget" name="scrollAreaWidgetContents">
+      <property name="geometry">
+       <rect>
+        <x>0</x>
+        <y>0</y>
+        <width>486</width>
+        <height>487</height>
```

Comment:
these values always bug me... I know they are auto-generated, but they often cause us to suddenly have much bigger minimum height of the widget

Reply:
tested and it doesn't seem to affect minimum height...